src/rinoh/template.py

```diff
@@ -561,7 +561,7 @@ def parse_string(cls, string):
 
     @classmethod
     def doc_repr(cls, value):
-        space = '\ ``\N{NO-BREAK SPACE}``\ '
+        space = r'\ ``\N{NO-BREAK SPACE}``\ '
```

Comment:
It does not work because of `\N`.
```suggestion
        space = '\\ ``\N{NO-BREAK SPACE}``\\ '
```